### PR TITLE
Runtime-Upgrade-MBA-3400

### PR DESCRIPTION
**Pull request by ibbar**
### Description

Please explain the changes this PR addresses here.

### Checklist

- [x] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by themacexpert — APPROVED** [review]
LGTM!

File: variables.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ networks:
     hex_chain_id: '0x507'
     chain_spec: alphanet
     block_explorer: https://moonbase.moonscan.io/
-    spec_version: runtime-3300
+    spec_version: runtime-3400
     parachain_release_tag: v0.41.0 # must be in this exact format for links to work
```
Comment (themacexpert) [inline variables.yml:19]:
```suggestion
    parachain_release_tag: v0.42.1 # must be in this exact format for links to work
```

File: variables.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ networks:
...
     parachain_sha256sum: d199266180adcdb25c19aa0ac9d8f5a7157ef1235393fac616d48bb68ba8bcde
     tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.41.0-3300-d3e6
```
Comment (themacexpert) [inline variables.yml:21]:
```suggestion
    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.42.1-3400-3ebd
```

File: variables.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ networks:
     hex_chain_id: '0x507'
     chain_spec: alphanet
     block_explorer: https://moonbase.moonscan.io/
-    spec_version: runtime-3300
+    spec_version: runtime-3400
     parachain_release_tag: v0.41.0 # must be in this exact format for links to work
     parachain_sha256sum: d199266180adcdb25c19aa0ac9d8f5a7157ef1235393fac616d48bb68ba8bcde
```
Comment (themacexpert) [inline variables.yml:20]:
```suggestion
    parachain_sha256sum: 9b645b2fd9e575b26ea727e96dc2a81486a73461bb961ba33ea2136e4c9060d8
```

**Review by themacexpert — CHANGES_REQUESTED** [review]

**Review by themacexpert — APPROVED** [review]

**Review by eshaben — COMMENTED** [review]
Were the links checked that use the `{{ networks.moonbase.spec_version }}` variable?

**Comment by themacexpert** [comment]
> Were the links checked that use the `{{ networks.moonbase.spec_version }}` variable?

Yup Ibra did

**Comment by eshaben** [comment]
Nice!! thanks!! 🚀

**Pull request merged**